third_party/kubernetes/pkg/kubectl/cmd/completion.go

```diff
@@ -0,0 +1,189 @@
+/*
+The original code is from https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/completion.go -
+
+Copyright 2016 The Kubernetes Authors.
```

Comment:
Add the modifications line below this one:
```
Modifications copyright 2018 the Heptio Ark contributors.
```

Reply:
fixed